index.d.ts

```diff
@@ -0,0 +1,28 @@
+interface ImageDataLike<T> {
+  width: number,
+  height: number,
+  data: T
+}
+
+type BufferRet = ImageDataLike<Buffer>;
+type UintArrRet = ImageDataLike<Uint8Array>;
+
+type ImageData = BufferRet | UintArrRet;
+type BufferLike = Buffer | Uint8Array | number[];
+
+export declare function encode(imgData: ImageDataLike<BufferLike>, qu: number): BufferRet;
```

Comment:
```suggestion
export declare function encode(imgData: ImageDataLike<BufferLike>, quality?: number): BufferRet;
```

quality is optional

Reply:
Ahh, yup. I shoulda caught this one before too haha